### PR TITLE
Create AppBaseTopBar and BaseTextField

### PR DESCRIPTION
**Pull request by manukyan-edgar**
Create components

### CODE REVIEW

File: app/src/main/java/com/simply/birthdayapp/presentation/ui/components/AppBaseTopBar.kt [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    Row(
+        modifier = Modifier
+            .height(48.dp)
+            .fillMaxWidth()
+            .background(BackgroundColor),
+        horizontalArrangement = Arrangement.SpaceBetween
+    ) {
+        IconButton(
+            modifier = Modifier
+                .padding(start = 16.dp),
```
Comment (Hayk985) [inline app/src/main/java/com/simply/birthdayapp/presentation/ui/components/AppBaseTopBar.kt:37]: We can keep modifiers with a single line if there is only one option set. Applies to other modifiers too.

File: app/src/main/java/com/simply/birthdayapp/presentation/ui/components/AppBaseTopBar.kt [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            onClick = onBackClick
+        ) {
+            Icon(
+                Icons.Filled.KeyboardArrowLeft,
+                "contentDescription",
```
Comment (Hayk985) [inline app/src/main/java/com/simply/birthdayapp/presentation/ui/components/AppBaseTopBar.kt]: Consider giving named arguments to make the code more readable. The same applies to other places too.

File: app/src/main/java/com/simply/birthdayapp/presentation/ui/components/BaseTextField.kt [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+import com.simply.birthdayapp.presentation.ui.theme.BackgroundColor
+import com.simply.birthdayapp.presentation.ui.theme.Primary2
+
+@Composable
+fun BaseTextField(
+    textState: String,
+    label: String = "",
+    onValueChange: (String) -> Unit = {},
+    keyboardType: KeyboardType = KeyboardType.Text,
+    shape: Shape = TextFieldDefaults.shape,
```
Comment (Hayk985) [inline app/src/main/java/com/simply/birthdayapp/presentation/ui/components/BaseTextField.kt:25]: There's a practice when we pass the lambda function arguments as the last one, so we can have the ability to open closures when we call this composable. This should work too if we use named arguments, but as a good practice consider it that way.

File: app/src/main/java/com/simply/birthdayapp/presentation/ui/components/AppBaseTopBar.kt (hunk 1 above)
Comment (manukyan-edgar) [reply app/src/main/java/com/simply/birthdayapp/presentation/ui/components/AppBaseTopBar.kt:37]: fixed

File: app/src/main/java/com/simply/birthdayapp/presentation/ui/components/AppBaseTopBar.kt (hunk 2 above)
Comment (manukyan-edgar) [reply app/src/main/java/com/simply/birthdayapp/presentation/ui/components/AppBaseTopBar.kt]: fixed

File: app/src/main/java/com/simply/birthdayapp/presentation/ui/components/BaseTextField.kt (hunk 3 above)
Comment (manukyan-edgar) [reply app/src/main/java/com/simply/birthdayapp/presentation/ui/components/BaseTextField.kt:25]: fixed

**Review by Hayk985 — APPROVED** [review]

**Pull request merged**